fiftyone/utils/caltech.py

```
@@ -0,0 +1,177 @@
+        logger.info("Using existing archive '%s'", archive_path)
+
+    logger.info("Extracting dataset...")
+    etau.extract_archive(archive_path)
+
+    _move_dir(dir_path, dataset_dir)
+
+    # There are two extraneous items in the raw download...
+
+    try:
+        etau.delete_dir(os.path.join(dataset_dir, "056.dog", "greg"))
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)